civismlext/test/test_stacking.py

```diff
@@ -827,6 +827,19 @@ def test_fit_pred_simple_clf():
                         np.array([[0, 1]]))
 
 
+def test_fit_params_regression(regression_test_data):
+    xtrain = regression_test_data['x']
+    ytrain = regression_test_data['y']
+    sample_weights = [1./len(ytrain)] * len(ytrain)
+    fit_params = {'rf__sample_weight': sample_weights}
+    sr = StackedRegressor([('rf', RandomForestRegressor(random_state=7)),
+                           ('rf2', RandomForestRegressor()),
+                           ('meta', NonNegativeLinearRegression())])
+    Xmeta, ymeta, _ = sr._base_est_fit_predict(
+        xtrain, ytrain, **fit_params)
+    assert Xmeta.shape == xtrain.shape
```

Comment:
This may be overkill, but it might be nice to have a classification case. But since this is a change to the base class that doesn't affect anything relating to targets `y`, I could be swayed that we can just stick with the regressor.

Reply:
Because it's a base class change that doesn't relate to the targets, I figured only one case was necessary. I can add the classifier to be safe.